### PR TITLE
M-44 (SAA) update

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Because the M-44 deserves more love.

## Why It's Good For The Game

> *ocelot meme here*

## Changelog
:cl:
add: Added the M-44 to more maps, usually replacing the CMB revolver at security armories.
add: Added gunslinger revolver speedloader for the M-44, which makes bullets ricochet to surfaces.
add: Added a unique gunsound to the M-44.
tweak: The M-44 is now chambered in special ricochet rounds, ammo available in certain maps.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -317,7 +317,7 @@
 	return current_mag ? current_mag.current_rounds : 0
 
 //-------------------------------------------------------
-//TP-44 COMBAT REVOLVER
+//TP-44 COMBAT REVOLVER, not based off on anything.
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/revolvers.dm:320]: Unneeded comment change

File: code/modules/projectiles/guns/revolvers.dm [hunk 2]
```diff
@@ -475,7 +476,7 @@
 	item_state = "mateba"
 
 //-------------------------------------------------------
-//MARSHALS REVOLVER
+//MARSHALS REVOLVER, leftover from CM.
```
Comment (TiviPlus) [inline code/modules/projectiles/guns/revolvers.dm:479]: Can this be fully deleted if yes please do so

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Remove the ricochet bullets those shouldn't be in normal gameplay although it's ok in eord

**Comment by XSlayer300** [comment]
What, the ricochet speedloaders? Also, I cannot change the specific ammo for the EORD weapon tiers.

> Remove the ricochet bullets those shouldn't be in normal gameplay although it's ok in eord

**Comment by TiviPlus** [comment]
Either way ricochet bullets shouldnt be in normal gameplay this way

**Comment by XSlayer300** [comment]
HGECK, GIMME TIME TO DO IRL STUFF FIRST.

**Comment by SilencedMP5A5** [comment]
i don't get what's wrong with the bouncy boolets

**Review by TheBimmer231 — APPROVED** [review]

**Comment by TiviPlus** [comment]
@XSlayer300 ?

**Pull request closed without merging**